<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataContainerReaderWidget</class>
 <widget class="QFrame" name="DataContainerReaderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>138</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="1">
    <widget class="QtSLineEdit" name="m_LineEdit"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="selectBtn">
     <property name="text">
      <string>Select File...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QColumnView" name="dcaProxyView">
     <property name="autoScroll">
      <bool>true</bool>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="resizeGripsVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Property</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="SVSmallLabel" name="absPathLabel">
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtSLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtSLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>SVSmallLabel</class>
   <extends>QLabel</extends>
   <header location="global">SVControlWidgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
